spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/CometColumnarBatchReader.java

```diff
@@ -43,7 +43,7 @@
  * populated via delegated read calls to {@link CometColumnReader VectorReader(s)}.
  */
 @SuppressWarnings("checkstyle:VisibilityModifier")
-class CometColumnarBatchReader implements VectorizedReader<ColumnarBatch> {
+public class CometColumnarBatchReader implements VectorizedReader<ColumnarBatch> {
```

Comment:
Why is this public now?

Reply:
Returned from a public method. IntelliJ complained about it.
Reverted as it is not touched after the new changes, and the PR is big enough as it is